CHANGELOG

```diff
@@ -1,3 +1,10 @@
+v1.2.0: 26-May-2021
```

Comment:
Should we also include some docs for how to use the flags and why you might want to in the docs section? https://github.com/hootsuite/vault-ctrl-tool/tree/master/docs

This is helpful but I like examples, I'm sure others would too.

Reply:
👍

Reply:
added some docs here docs/examples/force-refreshing-credentials.md